.github/workflows/CI.yml

```diff
@@ -19,7 +21,7 @@ jobs:
           - "nightly"
```

Comment:
Please remove line 18, the comment says it can be removed now! :)
```
          - "~1.9.0-0"  # To test heap snapshot; remove when v1.9 is latest Julia release.
```